migtests/scripts/run-test.sh

```diff
@@ -68,6 +68,14 @@ main() {
 	export_data
 	ls -l ${EXPORT_DIR}/data
 
+	if [ -f ${EXPORT_DIR}/logs/debezium.log ]
+	then
+		echo "Printing debeziun.log file"
+		cat ${EXPORT_DIR}/logs/debezium.log
```

Comment:
Please do not ignore comments :-) . This file is going to be huge. Do not output it on the GH console output! Just output last 100 lines. We usually don't need full file.

I would also suggest to output the log file snippet only when the test case fails.

Reply:
ohh sorry, i was going to take that comment, but i think before rebasing or something, i was unable to see this snippet in the code and thought this has been removed now. Sorry, will take this now.